app/views/main/index.html.erb

```diff
@@ -6,75 +6,25 @@
-          </div>
-        </div>
-      </section>
-
-      <section class="container my-4">
-        <div class="row">
-          <div class="col col-3">
-            <img src="https://dummyimage.com/250/9f9ede/000000" class="img-fluid" alt="user profile picture">
-          </div>
-          <div class="col col-8 border text-center align-middle m-1">
-           <p> Will contain username & number of posts <p>
-          </div>
-        </div>
-      </section>
-
-    <section class="container my-4">
-        <div class="row">
-          <div class="col col-3">
-            <img src="https://dummyimage.com/250/9f9ede/000000" class="img-fluid" alt="user profile picture">
-          </div>
-          <div class="col col-8 border text-center align-middle m-1">
-           <p> Will contain username & number of posts <p>
-          </div>
-        </div>
-      </section>
-
-      <section class="container my-4">
-        <div class="row">
-          <div class="col col-3">
-            <img src="https://dummyimage.com/250/9f9ede/000000" class="img-fluid" alt="user profile picture">
-          </div>
-          <div class="col col-8 border text-center align-middle m-1">
-           <p> Will contain username & number of posts <p>
-          </div>
-        </div>
-      </section>
-
-      <section class="container my-4">
-        <div class="row">
-          <div class="col col-3">
-            <img src="https://dummyimage.com/250/9f9ede/000000" class="img-fluid" alt="user profile picture">
-          </div>
-          <div class="col col-8 border text-center align-middle m-1">
-           <p> Will contain username & number of posts <p>
-          </div>
-        </div>
-      </section>
-
-      <section class="container my-4">
-        <div class="row">
-          <div class="col col-3">
-            <img src="https://dummyimage.com/250/9f9ede/000000" class="img-fluid" alt="user profile picture">
-          </div>
-          <div class="col col-8 border text-center align-middle m-1">
-           <p> Will contain username & number of posts <p>
-          </div>
-        </div>
-      </section>
-
-
-
-
+      <% if @users.blank? %>
+        <p>There are not any posts currently in the system.</p>
+      <% else %>
+        <% @users.each do |user| %>
+          <section class="container my-4">
+            <div class="row">
+              <div class="col col-3">
+                <img src="https://dummyimage.com/250/9f9ede/000000" class="img-fluid" alt="user profile picture">
+              </div>
+              <a href="/users/<%= user.id %>" class="col col-8">
+                <div class="username-text-container border text-center align-middle m-1">
+                  <p class="user-name-text"> <%= user.Name %> </p>
+                  <p class="posts-counter-text"> Number of Posts: <%= user.Posts_Counter %>  </p>
+                </div>
+              </a>
+            </div>
+          </section>
+        <% end %>
+      <% end %>
     </main>
```

Comment:
- [ ] [Optional] You should do well to put this code in your `users/index.html.erb` file for clarity

Reply:
Thanks for the suggestion